src/services/services.js

```diff
@@ -2,79 +2,87 @@
  * All API calls will be maintainted here
```

Comment:
What happend with ` https://api.spoonacular.com/recipes/complexSearch` request? It is crucial for searching by includings/excludings

Reply:
Hey Riserax I just added it under services -> getComplexSearch

Reply:
Ok, thanks!